### ISSUE TITLE
npa extractor

### ISSUE THREAD

**Issue by fisch123456** [Wilhansen/nipa#7]
Hi If I understand correctly this tool can open npa files right?Sorry if this is stupid question but I want to be sure.Thanks.

**Comment by fisch123456** [comment]
Probably don't know how to open npf files?

**Comment by Wilhansen** [comment]
Yeah, this is a tool to extract the contents of an NPA file as if it was a zip file.

Unfortunately, the archive changes per game release and the original author and I are no longer maintaining this.
